ansible/roles/stack-sunbird/templates/sunbird_player.env

```diff
@@ -95,3 +95,5 @@ sunbird_collectionEditorURL=collection-editor/index.html
 sunbird_contentEditorURL=content-editor/index.html
 sunbird_genericEditorURL=generic-editor/index.html
 sunbird_keycloak_public_key={{core_vault_sunbird_sso_publickey}}
+sunbird_bot_configured={{sunbird_bot_configured}}
```

Comment:
@rajeevsathish please keep a sane default value, if the bot is going to be inbuilt module keep it as `true`, else if it's an external module `false`